tests/wpt/web-platform-tests/XMLHttpRequest/responseurl.html

```diff
@@ -0,0 +1,21 @@
+<!doctype html>
+<html>
+  <head>
+    <title>XMLHttpRequest: responseURL test</title>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <link rel="help" href="https://xhr.spec.whatwg.org/#the-responseurl-attribute"/>
+  </head>
+  <body>
+    <div id="log"></div>
+    <script>
+      test(function() {
```

Comment:
We should also have a test that uses an HTTP redirect (resources/redirect.py) and ensures that responseURL shows the post-redirect URL.